masterlist.yaml

```diff
@@ -5012,6 +5012,78 @@ plugins:
   - name: 'Project Nevada - Med-X Fix.esp'
     tag: [ Names ]
 
+# Mothership Zeta Rehaul
+  - name: '(Mothership)?ZetaRehaul((BiWastelands|HardcoreZeta|(HardcoreZeta)?DelayDLCRedux|NVSpeechChecks)Patch)?\.es[mp]'
```

Comment:
While this regex is now functional, with multiple `?` included it becomes increasingly hard to read.
As such I still prefer the previous version so:
```yaml
  - name: '(MothershipZetaRehaul|ZetaRehaul(BiWastelands|DelayDLCRedux|HardcoreZeta(DelayDLCRedux)?|NVSpeechChecks)Patch)\.es[mp]'
```
It is basically a trade off between reducing the regex to a minimum state on the one side, and readability on the other side. Writing it like this also means, that the string `MothershipZetaRehaul` (so the name of the main plugin) is still intact, and as such easily searchable within the masterlist.